scalding-core/src/main/scala/com/twitter/scalding/TemplateSource.scala

```diff
@@ -1,121 +0,0 @@
-/*
-Copyright 2013 Inkling, Inc.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-package com.twitter.scalding
-
-import cascading.tap.hadoop.Hfs
-import cascading.tap.hadoop.{ TemplateTap => HTemplateTap }
-import cascading.tap.local.FileTap
-import cascading.tap.local.{ TemplateTap => LTemplateTap }
-import cascading.tap.SinkMode
-import cascading.tap.Tap
-import cascading.tuple.Fields
-
-/**
- * This is a base class for template based output sources
- */
-abstract class TemplateSource extends SchemedSource with HfsTapProvider {
```

Comment:
There was one instance of TemplateSource being used, which I've migrated to PartitionedSource.